.github/workflows/smoke_test.yml

```diff
@@ -21,3 +21,27 @@ jobs:
       - name: Run Tests
         run: |
           mix test
+
+  build_and_test_on_docker:
+    runs-on: ubuntu-22.04
```

Comment:
The `build_and_test` job above runs on `ubuntu-20.04`. While you're here, would you mind bumping it?